projects/plugins/backup/src/js/hooks/useConnection.js

```diff
@@ -72,8 +72,11 @@ export default function useConnection() {
 				isUserConnected={ connectionStatus.isUserConnected }
 				apiRoot={ APIRoot }
 				apiNonce={ APINonce }
+				currentPlugin="jetpack-backup"
```

Comment:
I wonder if the string "jetpack-backup" is set somewhere more globally for the plugin that we might be able to use here.

Reply:
There's one now: https://github.com/Automattic/jetpack/tree/master/projects/js-packages/config

It would be awesome if we used that directly on the `ConnectionStatusCard`, no need to pass it via prop.

See an example here: https://github.com/Automattic/jetpack/pull/21672

Reply:
Updated this and the base PR to use this new method!